hunk.js

```diff
@@ -203,7 +203,7 @@ function makeHunks (changes, precontext, postcontext) {
         if (change.type === UNMODIFIED) {
             // add context
             let ctx_after  = ci > 0 ? postcontext : 0              // context lines following previous change
-            let ctx_before = ci < changes.length ? precontext : 0  // context lines preceding next change (iff there are more changes)
+            let ctx_before = ci < changes.length - 1? precontext : 0  // context lines preceding next change (iff there are more changes)
```

Comment:
Slightly strange to leave the 1 right next to the '?'

Reply:
Need to update the test.  The tests cover this case (and currently assert the wrong behavior.) test should be updated to this:

<img width="1041" alt="Screen Shot 2019-05-09 at 1 16 02 AM" src="https://user-images.githubusercontent.com/6353604/57428850-1c2c3c80-71f8-11e9-8caa-9a624b1a696d.png">

Reply:
After making the update to the test, run the tests by going into the directory and doing:

npm install 
npm test

Make sense?

Reply:
Are you talking about the format?

Reply:
Hello,I have modified the format issue.